CodenameAlphamale/FaultWithCuriosity
# Button random selection

How to test:
1. Start a host and a client with ParallelSync
2. Check for changes on the note prefab

Puzzle1, Puzzle2 and Parkour puzzles has one script attached to the a Note prefab (several for Puzzle2) 
- "RandomizerCode" script <--check for the NetworkVariable

Puzzle3 has a ButtonPuzzle prefab with 2 attached scripts
- "PuzzleInputController"
- "Chose One Button" script <--check for the NetworkVariable

Check changes are consistent on both host and client

3. Try a puzzle, probably Puzzle1 since it is close
4. ...
5. profit
